Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -6607,6 +6607,35 @@ script:
     - contextPath: Panorama.Spyware.BotentDomain.Max_version
       description: The botnet domain max version.
       type: String
+  - arguments:
+      - name: Anti_spyware_profile_name
+        description: The name of the anti spyware profile. if the profile exists will operate on it. else, if a new name is given a new Anti-Spyware Profile will be created.
```

Comment:
```suggestion
        description: The name of the anti spyware profile. if the profile exists, the command will operate on it, Otherwise, if a new name is given a new Anti-Spyware Profile will be created.
```